data/UrlStore.js

```diff
@@ -5,14 +5,13 @@
  * Copyright © 2020 Extremely Heavy Industries Inc.
  */
 
-import {LoadSupport, XH} from '@xh/hoist/core';
+import {XH} from '@xh/hoist/core';
 
 import {Store} from './Store';
 
 /**
  * A store with built-in support for loading data from a URL.
  */
-@LoadSupport
```

Comment:
This ends up being quite nice, that you can simply implement `doLoadAsync()` to enable all the LoadSupport functionality.